libr/anal/anal.c

```diff
@@ -64,6 +64,8 @@ static void zign_rename_for(REvent *ev, int type, void *user, void *data) {
 void r_anal_hint_storage_init(RAnal *a);
 void r_anal_hint_storage_fini(RAnal *a);
 
+R_IPI void r_meta_item_free(void *_item);
```

Comment:
does it need `R_IPI`? It seems to be used only in this file, `static`?

Reply:
Indeed, it can be moved.